GapBackfill/.vscode/extensions.json

```diff
@@ -3,4 +3,4 @@
     "ms-azuretools.vscode-azurefunctions",
     "ms-vscode.csharp"
```

Comment:
maybe we should exclude the whole .vscode folder because it will be differently for each consumer anyways